### DISCUSSION TITLE
Using same model for multiple resources

### DISCUSSION THREAD

**Discussion by ayranxs** [filamentphp/filament#5073] [Help]
When you create two resources with different labels for same model, it treats as one resource.

For example; when you have Orders model and instead of just showing Orders menu then letting user to use filters to get Pending/Completed orders, you may want to leverage getEloquentQuery on resources to automatically filter out by order state. So this way, you can have Pending Orders / Completed Orders menu.

In Nova, you can create multiple resource for same model, then you can give each other different label and voila; they're added to navigation

### Expected behavior

For example, when you have Order model and you created PendingOrdersResource with custom $label (Pending Orders) & CompletedOrdersResource with custom $label (Completed Orders), they should be listed on the navigation as two seperate resources. Instead you'll see two Orders menu on the navigation, when you click on each other, you'll see that the URI is same on them (/admin/orders)

How can i use same model for multiple resources like Laravel Nova?

**Comment by RX-77IT** [comment]
You can change the url [with $slug](https://filamentphp.com/docs/2.x/admin/resources/getting-started#customizing-the-url-slug) in the Resource file.
```php
protected static ?string $slug = 'pending-orders';
```
Then I think you need to [change the eloquent query](https://filamentphp.com/docs/2.x/admin/resources/getting-started#customizing-the-eloquent-query) that retrieves the records using getEloquentQuery() in the Resource file.
```php
public static function getEloquentQuery(): Builder
{
    return parent::getEloquentQuery()->where(...); // query to retrieve only pending orders
}
```
